### ISSUE TITLE
Turbo frames reset focus

### ISSUE THREAD

**Issue by ashton22305** [OSC/ondemand#4945]
When the system status or active jobs pages are updated, the focus is reset. This prevents users with screen readers from reading these pages. We must fix this in order to meet [WCAG SC 1.3.1 Info and Relationships](http://www.w3.org/TR/WCAG20/#content-structure-separation-programmatic)

**Comment by Bubballoo3** [comment]
I am unable to duplicate in either ActiveJobs or SystemStatus. In both cases, keyboard focus appears unaffected when the turbo frames reload (though this may not be the case if the focus lies within the updated element)

**Comment by Bubballoo3** [comment]
Was able to confirm that this is an issue when focus is within the turbo frame

**Comment by johrstrom** [comment]
I believe this was fixed in #5079. At least for system status - active jobs doesn't use turbo frames but could still reset focus.
